### PR TITLE
v5.23.2 Changelog

### PR DESCRIPTION
**Pull request by amyblais**
Dot release docs

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -6,7 +6,10 @@ Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
 ## Release v5.25 - [ESR](https://docs.mattermost.com/administration/release-definitions.html#extended-support-release-esr)
 
-**Release day: 2020-07-16**
+- **v5.25.1, released **
+  - Fixed an issue where LDAP sync sent cache clear messages in a tight loop. [MM-27114](https://mattermost.atlassian.net/browse/MM-27114)
```
Comment (amyblais) [inline source/administration/changelog.md:10]: Unsure if the description needs any more details.

Comment (agnivade) [reply source/administration/changelog.md:10]: I will send an updated description once I am able to reproduce the issue more clearly.

Comment (agnivade) [reply source/administration/changelog.md:10]: There are 3 fixes now:
- Smoothen the DB query load while syncing teams and channel roles by fetching data in batches.
- Fix a bug in pagination which would query more data redundantly.
- Throttle network traffic by implementing bounded concurrency.

File: source/administration/changelog.md [hunk 2]
```diff
@@ -6,7 +6,12 @@ Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
 ## Release v5.25 - [ESR](https://docs.mattermost.com/administration/release-definitions.html#extended-support-release-esr)
 
-**Release day: 2020-07-16**
+- **v5.25.1, released 2020-07-23**
+  - Smoothened the database query load while syncing teams and channel roles by fetching data in batches. [MM-27114](https://mattermost.atlassian.net/browse/MM-27114)
```
Comment (justinegeffen) [inline source/administration/changelog.md:10]:
```suggestion
  - Smoothed the database query load while syncing teams and channel roles by fetching data in batches. [MM-27114](https://mattermost.atlassian.net/browse/MM-27114)
```

File: source/administration/changelog.md [hunk 3]
```diff
@@ -80,6 +85,10 @@ Multiple setting options were added to `config.json`. Below is a list of the add
 
 ## Release v5.24 - [Feature Release](https://docs.mattermost.com/process/release-faq.html#release-overview)
 
+- **v5.24.3, released 2020-07-23**
+  - Smoothened the database query load while syncing teams and channel roles by fetching data in batches. [MM-27114](https://mattermost.atlassian.net/browse/MM-27114)
```
Comment (justinegeffen) [inline source/administration/changelog.md:89]:
```suggestion
  - Smoothed the database query load while syncing teams and channel roles by fetching data in batches. [MM-27114](https://mattermost.atlassian.net/browse/MM-27114)
```

File: source/administration/changelog.md [hunk 4]
```diff
@@ -239,6 +248,10 @@ Multiple setting options were added to `config.json`. Below is a list of the add
 
 ## Release v5.23 - [Quality Release](https://docs.mattermost.com/process/release-faq.html#release-overview)
 
+- **v5.23.2, released 2020-07-23**
+  - Smoothened the database query load while syncing teams and channel roles by fetching data in batches. [MM-27114](https://mattermost.atlassian.net/browse/MM-27114)
```
Comment (justinegeffen) [inline source/administration/changelog.md:252]:
```suggestion
  - Smoothed the database query load while syncing teams and channel roles by fetching data in batches. [MM-27114](https://mattermost.atlassian.net/browse/MM-27114)
```

**Comment by justinegeffen** [comment]
Non-blocking: I suggested "smoothed" to replace "smoothened" as it reads better. If "smoothened" is the technical term then disregard my comments. :)

**Pull request merged**